service/controller/v2/resources/cleaning/resource.go

```diff
@@ -0,0 +1,50 @@
+package cleaning
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	Name = "cleaningv2"
+)
+
+func (r *Resource) Name() string {
+	return Name
+}
+
+// Config represents the configuration used to create a new cleansing resource.
+type Config struct {
+	G8sClient versioned.Interface
+	K8sClient kubernetes.Interface
+	Logger    micrologger.Logger
+}
+
+type Resource struct {
```

Comment:
Is this a permanent or temporary resource? What is the problem we try to solve?

Reply:
Some installation contains certconfigs that belongs to deleted TCs. 
In `geckon` you could see as below. 
```
k get certconfigs
...
6srrb-app-operator-api       135d
6srrb-cluster-operator-api   135d
8tjv9-app-operator-api       174d
d20if-app-operator-api       170d
fzcf0-app-operator-api       103d
fzcf0-cluster-operator-api   103d
n00mo-app-operator-api       147d
```
After we deleted all these remaning certconfigs, we can safely revert this change.